wlcreate/Start-Asian-Love
# New allyship

## Description

- Resources specific to white people have been added to aid in allyship with AAPI and BIPOC communities.
- Resources and the new allyship can definitely be reviewed, discussed, and edited. 
- There are many 2020 white-specific allyship articles in relation to the BLM movement, but white-specific allyship articles in relation to AAPI Communities are currently lacking and/or outdated in the media.
- Currently, "education" is the only category for the white allyship resources.
- New representation entries have been added to the Wins page. Lisa Ling is the first writer/journalist representative.
- Running `npm install` prompted an update for the `Next` dependency.

Related Issue: #15 





Resources:
- Jasmine M. Cho TED Talk: https://www.youtube.com/watch?v=DwTKK5WCAK0
- A great AAPI-centric perspective of history that could possibly be incorporated in the future: https://aatimeline.com/



## Acceptance Criteria

- Create categories for:
   - Education
   - Attacks
   - Financial
   - Rights
   - Volunteer
   - Health
   - (optionally) Art
- Create allyships for:
   - Asians/AAPI
   - Everyone
   - BIPOC
   - **New Allyship** for white people
- Allow users to toggle between the different allyships and categories
- When the users toggle, only the related resources are displayed



## Self-check

Please check all that apply:



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



- Wins Page: Representation
![Start Asian Love _ Wins (1)](https://user-images.githubusercontent.com/61435324/119853590-57346200-bede-11eb-8000-25f07a19037a.gif)


- Support Page Resources
![Start Asian Love _ Support (4)](https://user-images.githubusercontent.com/61435324/119853560-4f74bd80-bede-11eb-80a5-6a08a4bde3c5.gif)

![Start Asian Love _ Support (1)](https://user-images.githubusercontent.com/61435324/119853685-69ae9b80-bede-11eb-95a0-1ad80153b38c.gif)

### After

- Wins Page: Representation
![Start Asian Love _ Wins](https://user-images.githubusercontent.com/61435324/119853963-a7abbf80-bede-11eb-8984-24f545efa8fc.gif)

- Support Page Resources
![Start Asian Love _ Support (3)](https://user-images.githubusercontent.com/61435324/119854010-b1cdbe00-bede-11eb-80f5-93eb068e1388.gif)






## Testing Steps / QA Criteria

- Switch to branch: `git checkout new-allyship`
- Install dependencies: `npm install`
- In Support Page, click white allyship button to filter resources. Click cards, check for any typos, and check links.
- In Wins Page, click representation button to see new entries. Click cards, check for any typos, and check links.
